doc/source/diagnostics/libcoredumper.rst

```diff
@@ -0,0 +1,95 @@
+
+The documented moment of a computer when either the computer or an application crashed is a core dump file. Developers examine the dump find the cause of the failure. 
+
+The ``libcoredumper`` is a free and Open Source implementation of the ``google-coredumper``, enhanced to work on newer Linux versions, and GCC and CLANG. 
+
+You should test before putting this tool into production. We have tested the tool on each of the currently supported operating systems.
+
+.. rubric:: Enabling the ``libcoredumper``
+
+Enable core dumps for troubleshooting purposes. 
+
+To enable the ``libcoredumper``, add the ``coredumper`` variable to the ``mysqld`` section of ``my.cnf``. This variable is independent of the older ``core-file`` variable. 
+
+The variable can have the following possible values: 
+
+.. tabularcolumns:: |p{5cm}|p{11cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value 
+     - Description
+   * - Blank
+     - The core dump is saved under MySQL datadir and named ``core``.
+   * - A path ending with /
+     - The core dump is saved under the specified directory and named ``core``.
+   * - Full path with a filename
+     - The core dump is saved under the specified directory and with the specified filename
+
+Restart the server.
+
+.. rubric:: Verifying the ``libcoredumper`` is Active
+
+MySQL writes to the log when generating a core file and delegates the core dump operation to the Linux kernel. An example of the log message is the following:
+
+.. code-block:: text
+
+    Writing a core file
+
+MySQL using the ``libcoredumper`` to generate the file creates the following message in the log:
+
+.. code-block:: text
+
+    Writing a core file using lib coredumper
+
+Every core file adds a crash timestamp instead of a PID for the following reasons:
+
+* Lets you correlate the core file with a crash. MySQL prints a UTC timestamp on the crash log.
+
+    .. code-block:: bash
+
+        10:02:09 UTC - mysqld got signal 11;
+
+
+* Lets you keep multiple core files. 
+
+.. note:: 
+
+    For example, operators and containers run as PID 1. If the process ID identified the core file, each container crash generates a core dump that overwrites the previous core file.
+
+The libcoredumper releases a stack trace also adds the following: 
+
+.. tabularcolumns:: |p{5cm}|p{11cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name 
+     - Description
+   * - Prints binary BuildID
+     - The Strip utility removes unneeded sections and debugging information to reduce the size. This method is standard with containers where the size of the image is essential. The BuildID lets you resolve the stack trace when the Strip utility removes the binary symbols table.
+   * - Print the server version information
+     - The version information establishes the starting point for analysis. Some applications, such as MySQL, only print this information to a log on startup, and when the crash occurs, the size of the log may be large, rotated, or truncated.  
```

Comment:
I am not sure this part belongs to this documentation. We may add this to a new page as this works independently of libcoredumper, for example, even though you have not configured coredumper variable, PS will print buildID and server version to the crash information.

Reply:
I'll need more information.

Reply:
OK. After discussion, I'll remove this and make it a different ticket.